Development/nmos-cpp-node/node_implementation.cpp

```diff
@@ -894,6 +895,17 @@ void node_implementation_init(nmos::node_model& model, slog::base_gate& gate)
         auto channelmapping_output = nmos::make_channelmapping_output(id, name, description, source_id, channel_labels, routable_inputs);
         if (!insert_resource_after(delay_millis, model.channelmapping_resources, std::move(channelmapping_output), gate)) throw node_implementation_init_exception();
     }
+
+    // example root block
+    auto root_block = nmos::make_root_block();
+    // example device manager
+    auto device_manager = nmos::make_device_manager(2, root_block, model.settings);
```

Comment:
Is a constant better here than a "2"?

Reply:
This is the `oid` for the device manager instance, which can be between `2-65535`, as the root block can only be `1`

Reply:
In the short term, perhaps assign the value to a temporary variable called `oid`, or `device_manager_oid` to mitigate the "magicness" of the raw number?

In the long term we need to work out a strategy for assigning oids to objects. For instance a function that returns a monotonically increasing integer value for automatically generating the oids.

Reply:
Will leave this to implementers to decide on strategy for managing oids